sams-config.plugin.zsh

```diff
@@ -2,37 +2,29 @@ alias desk='cd ~/Desktop && ls'
 alias doc='cd ~/Documents && ls'
 alias down='cd ~/Downloads && ls'
 alias proj='cd ~/proj && ls'
-alias prj='cd ~/proj && ls'
-alias zrc='vim ~/.zshrc'
-alias vrc='vim ~/.vimrc'
-alias erc='vim ~/.eslintrc'
+alias zrc='mvim -v ~/.zshrc'
+alias vrc='mvim -v ~/.vimrc'
 alias cl='clear'
-alias m='vim'
 alias colours='spectrum_ls'
-alias st='open -a SourceTree .'
 alias n='npm run'
 alias lll='ls' # for typing ls with one hand
 alias pg='postgres -D /usr/local/var/postgres'
 alias ss='echo "sourcing zshrc"; source ~/.zshrc'
 alias ns='npm start'
 alias nt='npm t'
 alias nw='npm run watch'
-alias nsm='nodemon -e js,json,html,css -x "npm start"'
-alias firefox='/Applications/Firefox.app/Contents/MacOS/firefox'
-alias nc='npm run coverage'
-alias ntm="nodemon -x 'npm t'"
-alias opencoverage='open coverage/lcov-report/index.html'
-alias za='vim ~/.oh-my-zsh/plugins/sams-aliases/sams-aliases.plugin.zsh'
-alias ric='cd ~/proj/crm-base-generic/'
-alias le='cd ~/proj/le'
-alias lang='cd ~/proj/lang'
-alias gito='cd ~/proj/gitodoro'
-alias gitod='cd ~/proj/gitodoro'
-alias gitodo='cd ~/proj/gitodoro'
-alias gitodor='cd ~/proj/gitodoro'
-alias gitodoro='cd ~/proj/gitodoro'
-alias mc='cd ~/proj/my-config'
+alias za='mvim -v ~/.oh-my-zsh/plugins/sams-config/sams-config.plugin.zsh'
+alias ze='mvim -v ~/.zsh_envs'
 alias m='mvim -v'
+alias ms='mix phx.server'
+alias mpr='mix phx.routes'
+alias python=python3
+alias pip=pip3
+alias dj='python manage.py'
+alias ggpuhs=ggpush
```

Comment:
I assume that this is just to catch a typo you often make?

Reply:
exactly 👍